tests/integration/components/settings/application-page-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('settings/application-page', 'Integration | Component | settings/application page', {
```

Comment:
@sumedh123 

See how the integration test of another component is defined -> https://github.com/fossasia/open-event-frontend/blob/development/tests/integration/components/event-card-test.js 

See 
```
import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
```

See 
```
moduleForComponent('event-card', 'Integration | Component | event card');
```